castle/cms/profiles/2_3_3/catalog.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<object name="portal_catalog" meta_type="Plone Catalog Tool">
+
+ <index name="recurrence" meta_type="FieldIndex">
```

Comment:
I think it's only metadata that you are trying to store right?

I think you can remove `<index>` and just leave `<column>`.

Reply:
yup, seems to work fine. thx!